samples/highcharts/series-on-point/series-on-point/demo.js

```diff
@@ -0,0 +1,62 @@
+Highcharts.chart('container', {
+
+    plotOptions: {
+        pie: {
+            minSize: 25,
+            maxSize: '20%',
+            dataLabels: {
+                enabled: false
+            }
+        }
+    },
+
+    title: {
+        text: 'Series on point'
+    },
+
+    subtitle: {
+        text: 'Pies displayed over points with a given id'
+    },
+
+    series: [{
+        type: 'column',
+        data: [
+            { y: 3, id: 'firstPoint' },
+            { y: 5, id: 'secondPoint' },
+            { y: 5, id: 'thirdPoint' },
+            3,
+            2,
+            3
+        ]
```

Comment:
When toggling the series with the legend, the pies remain while the columns disappear. In the map-pie demo I see that the pies do disappear. Is this intentional, or should it disappear along with the columns?

Reply:
It should ;) I'll fix it, thanks